From e4dfe018323e1ddd6e8c405b49e6124b90571338 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Sat, 6 Aug 2011 00:20:58 +0200
Subject: [PATCH 8/8] File access: rename the callbacks

This is needed for a reason that I don't get at all...
This is probably because of some mangling of static modules...
---
 modules/access/file.c |    4 ++--
 modules/access/fs.c   |    2 +-
 modules/access/fs.h   |    4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules/access/file.c b/modules/access/file.c
index c89a6ae..0c82728 100644
--- a/modules/access/file.c
+++ b/modules/access/file.c
@@ -134,7 +134,7 @@ static bool IsRemote (int fd)
 /*****************************************************************************
  * Open: open the file
  *****************************************************************************/
-int Open( vlc_object_t *p_this )
+int Activate( vlc_object_t *p_this )
 {
     access_t     *p_access = (access_t*)p_this;
 #ifdef WIN32
@@ -258,7 +258,7 @@ error:
 /*****************************************************************************
  * Close: close the target
  *****************************************************************************/
-void Close (vlc_object_t * p_this)
+void Deactivate (vlc_object_t * p_this)
 {
     access_t     *p_access = (access_t*)p_this;
 
diff --git a/modules/access/fs.c b/modules/access/fs.c
index 8f01ea7..bb0e5b7 100644
--- a/modules/access/fs.c
+++ b/modules/access/fs.c
@@ -70,7 +70,7 @@ vlc_module_begin ()
     add_obsolete_string( "file-cat" )
     set_capability( "access", 50 )
     add_shortcut( "file", "fd", "stream" )
-    set_callbacks( Open, Close )
+    set_callbacks( Activate, Deactivate )
 
     add_submodule()
     set_section( N_("Directory" ), NULL )
diff --git a/modules/access/fs.h b/modules/access/fs.h
index de5e493..cedb402 100644
--- a/modules/access/fs.h
+++ b/modules/access/fs.h
@@ -20,8 +20,8 @@
 
 #include <dirent.h>
 
-int Open (vlc_object_t *);
-void Close (vlc_object_t *);
+int Activate (vlc_object_t *);
+void Deactivate (vlc_object_t *);
 int NoSeek (access_t *, uint64_t);
 
 ssize_t FileRead (access_t *, uint8_t *, size_t);
-- 
1.7.4.1

